docs/api/sql/Raster-operators.md

```diff
@@ -676,7 +676,7 @@ Introduction: Add a band to a raster from an array of doubles.
 
 Format: `RS_AddBandFromArray (raster: Raster, band: Array[Double])` | `RS_AddBandFromArray (raster: Raster, band: Array[Double], bandIndex:Int)` | `RS_AddBandFromArray (raster: Raster, band: Array[Double], bandIndex:Int, noDataValue:Double)`
 
-Since: `v1.4.1`
+Since: `v1.5.0`
```

Comment:
You can separate the new APIs and old APIs by 

```
Additional APIs since `v1.5.0`
XXXX
```

Reply:
This change is not version dependent, if a user loads a raster with some other data type, and tries to add a band, they will be typecasted.
However, I changed the version because 1.5.0 onwards MakeEmptyRaster allows specifying data type.

Also, once we can control supplying version's docs from the backend right, I didn't think I would need to specify version wise anything in the documentation